app/bundles/LeadBundle/Assets/js/lead.js

```diff
@@ -379,6 +379,9 @@ Mautic.addLeadListFilter = function (elId) {
         if (fieldType == 'boolean') {
             templateField = 'select';
         }
+        if (fieldType == 'multiselect') {
```

Comment:
Let's combine this with line 379 `if (fieldType == 'boolean' || fieldType == 'multiselect') {`